### ISSUE TITLE
No API documentation

### ISSUE THREAD

**Issue by steve-biggs-fox** [CATIA-Systems/FMPy#660]
There is no API documentation. There is a basic tutorial that shows how to print FMU info, run it and plot the results, but that is all. For anything else, it just says to see the examples. Really, this needs API documentation to say what functions are available, how to call them, what is returned, etc. For example, how do I run an FMU with non-default parameters? What is this SimulationResult object that is returned? How do I access specific output variables? etc. Sure, I might be able to work it out from the examples, but that is long and annoying. API documentation would get straight to the point. And sure, I can type help(function_name), but then I have to use auto-complete to see what's available, remember where all the different functions are, etc. which is difficult when you're new to a package. API documentation would provide an overview of everything for reference while learning. Plus, I think it would be easy to create since you already have the docstrings - you just need a package to parse that and auto-generate the docs (many such packages exist). Please consider creating API documentation. Thank you

Other than that, from what I've seen so far, this seems like a great package, so thanks for creating it :)

**Comment by moonglaivecn** [comment]
As a beginner, I also really need the API documentation. Reading the source code to understand how each method is used is indeed quite inefficient.

**Comment by Redlargeforest** [comment]
If only it were like pyfmi, with full api documentation, but pyfmi doesn't support the fmi 3.0 standard

**Comment by Redlargeforest** [comment]
> > If only it were like pyfmi, with full api documentation [...]
> 
> Do you refer to https://jmodelica.org/pyfmi/?

Yes, I don't seem to see the 3.0 related introduction, it only stays in 2.0, the new fmi3.0 they don't seem to update the documentation.
